metrics/cli/docs/features/update/06_update_feature.md

```diff
@@ -0,0 +1,139 @@
+# Update feature
+> Feature description / User story.
+
+As a user, I want to have the opportunity to update an existing project, so I don't have to create a new one every time.
+
+## Contents
+
+- [**Analysis**](#analysis)
+    - [Feasibility study](#feasibility-study)
+    - [Requirements](#requirements)
+    - [Landscape](#landscape)
+    - [Prototyping](#prototyping)
+    - [System modeling](#system-modeling)
+
+# Analysis
+> Describe a general analysis approach.
+
+During the analysis stage, we are going to investigate the implementation approaches for updating existing projects with
+the Metrics CLI.
+
+### Feasibility study
+> A preliminary study of the feasibility of implementing this feature.
+
+The `Update feature` expands the end-user capabilities and improves the UX of the Metrics CLI, so this feature
+makes sense for the project.
+
+Moving on to the feasibility of implementation, the `Update feature` requires the following actions:
+
+- parsing the YAML configuration file;
+- redeploying/updating the existing Metrics Web application to the same Firebase project.
+
+The following proofs cover the above requirements, so we can implement the `Update feature`:
+- to parse the YAML configuration file, we can use a [package](https://github.com/Flank/flank-dashboard/tree/master/yaml_map) that applies in the ci_integrations module for the same purposes;
+- the Metrics CLI has everything we need to perform the redeploy action to the same Firebase project since the CLI can perform a deploy action.
+
+### Requirements
+> Define requirements and make sure that they are complete.
+
+Let's start with visiting and analyzing resources to collect the necessary data during the analysis stage.
+Thanks to these data, we can create precise and determined requirements and edge cases for the `Update feature`.
+
+The `Update feature` has the following requirements:
+- should read the configuration from the YAML configuration file;
+- should consume the configuration file path via the command line arguments;
+- should validate the YAML configuration file;
+- should stop the redeploy process of the Metrics CLI if something went wrong during the process;
+- should redeploy/update the existing Metrics Web application to the same Firebase project;
+- configuring the Sentry should be optional.
```

Comment:
I suppose it would be great to divide the list into two different lists that would provide requirements for different actions. This allows making the requirements clearer. What do you think? 

Also, that would be great to tell something about edge cases. For example, if the updating fails, what would happen to the currently deployed version of Metrics components? And what should happen? Should the user be able to add Sentry to the deployed project while updating? And so on.